source/cloud/vault/connecting.md

```diff
@@ -1,4 +1,5 @@
 ```eval_rst
+.. title: UKFast Documentation | eCloud Vault | Connecting to eCloud Vault
```

Comment:
If this is to set the actual title of the page then you can just edit this line:

`# Connecting to eCloud Vault`